charts/agents-graphql-service/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+type: application
+appVersion: 1.0.0
+description: STRG.BeHave agents-graphql-service
```

Comment:
its agents-graphql-api or?

Reply:
yes, has been renamed durring github migration, would be good to adjust

Reply:
@jazzlyn  I renamed whole service to api